README.md

```diff
@@ -48,29 +51,29 @@ Example: `edge://?[params…]` or `https://deep.edge.app/reqaddr?[params...]`
 
```

Comment:
You still need to change the `edge://?[params…]` example on the line above to `edge://reqaddr?[params…]`.

Reply:
Oops. I had that in an earlier edit and missed adding it before redoing changes due to a conflict

Reply:
rebased and made the fix